core/src/main/resources/hudson/Messages_fr.properties

```
@@ -49,16 +47,82 @@ Util.day   ={0} j
+ Veuillez consulter la note de livraison du plugin pour plus de détails.
+PluginManager.parentDepCompatWarning=Les plugins suivants sont incompatibles :
+PluginManager.parentCompatWarning=Les plugins suivants sont affectés par cela :
+PluginManager.coreWarning=\
+ Avertissement : ce plugin est compatible pour Jenkins {0} ou plus récent. \
+ Jenkins ne démarrera pas ce plugin s''il est installé.
+PluginManager.depCompatWarning=\
+ Avertissement : ce plugin nécessite des versions plus récentes de dépendances et au moins un de ces plugins n''est pas compatible avec la version installée. \
+ C''est habituellement le cas parce que son comportement ou ses APIs ont changé, ou parce qu''il utilise un format de configuration différent de celui installé. \
+ D''autres plugins ayant une dépendance à ce plugin sont peut-être incompatibles avec cette mise à jour et ne fonctionneront plus comme attendu, les jobs utilisant ce plugin necessiteront peut-être d''être reconfigurés, et/ou vous ne serez peut-être plus capable de revenir à la version précédente sans avoir à restaurer manuellement les configurations précédentes. \
+ Veuillez consulter la note de livraison du plugin pour plus de détails.
+PluginManager.depCoreWarning=\
+ Avertissement : ce plugin possède des dépendances à d''autres plugins qui requièrent Jenkins {0} ou plus récent. \
+ Jenkins refusera de charger les dépendances nécessitant une version plus récente de Jenkins, \
+ et en conséquence ce plugin échouera à se charger.
+PluginManager.securityWarning=\
+  Avertissement : cette version de plugin n''est peut-être par sûre. Veuillez vérifier les notices de sécurité suivantes :
+PluginManager.ago=Il y a {0}
+PluginManager.adoptThisPlugin=\
+  <strong>Ce plugin est à adopter!</strong> Nous recherchons de nouveaux mainteneurs. \
+  Veuillez consulter notre initiative <a href="https://www.jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/" rel="noopener noreferrer" target="_blank">Adopter un plugin</a> pour plus d''informations.
+PluginManager.deprecationWarning=<strong>Ce plugin est obsolète.</strong> Généralement, cela signifie qu''il est soit obsolète, plus développé, ou ne fonctionne peut-être plus. <a href="{0}" rel="noopener noreferrer" target="_blank">En apprendre plus.</a>
+PluginManager.insecureUrl=\
+  Vous utilisez une URL non sécurisée pour télécharger le plugin, utilisez-la à vos propres risques et périls!
+PluginManager.invalidUrl=\
+  Vous utilisez une URL invalide pour télécharger le plugin. Les URLs file, https et http (non recommandé) sont supportées.
+
+PluginManager.emptyUpdateSiteUrl=\
+  Le site de mise à jour ne peut être vide. Veuillez entrer une URL valide.
```

```suggestion
  Le site de mise à jour est anormalement vide. Veuillez entrer une URL valide.
```
Subjective, seems more easier to understand that way.